task/attest/build.rs

```diff
@@ -60,5 +67,24 @@ pub const ALIAS_DATA: DataRegion = DataRegion {{
         region.address, region.size
     )?;
 
+    // Get list of those tasks allowed to reset the attestation log
+    if let Ok(task_config) = build_util::task_config::<TaskConfig>() {
+        writeln!(
+            out,
+            "pub const PERMIT_LOG_RESET: [u16; {}] = [",
+            task_config.permit_log_reset.len()
+        )?;
+        let tasks = build_util::task_ids();
+        for task_name in task_config.permit_log_reset {
+            let id = tasks.get(task_name.as_str()).context(format!(
```

Comment:
Nit: it's typical to use `with_context` if the context operation does work (like formatting a string).
```suggestion
            let id = tasks.get(task_name.as_str()).with_context(|| format!(
```

Reply:
Thanks